sympy/stats/crv_types.py

```diff
@@ -2849,14 +2850,21 @@ class NormalDistribution(SingleContinuousDistribution):
 
     @staticmethod
     def check(mean, std):
-        _value_check(std > 0, "Standard deviation must be positive")
+        _value_check(std >= 0, "Standard deviation must be nonnegative.")
 
     def pdf(self, x):
-        return exp(-(x - self.mean)**2 / (2*self.std**2)) / (sqrt(2*pi)*self.std)
+        # https://en.wikipedia.org/wiki/Normal_distribution#Zero-variance%20limit
+        zero_var_pdf = DiracDelta(x - self.mean, H0=1)
```

Comment:
```suggestion
        zero_var_pdf = DiracDelta(x - self.mean)
```
This argument is completely ignored, so probably better to not pass it.